cs/src/comm/epoxy-transport/Frame.cs

```diff
@@ -48,10 +50,12 @@ public Framelet(FrameletType type, ArraySegment<byte> contents)
 
         public static bool IsKnownType(UInt16 value)
         {
-            return value == (UInt16)FrameletType.LayerData
-                | value == (UInt16)FrameletType.PayloadData
-                | value == (UInt16)FrameletType.EpoxyHeaders
+            return false
```

Comment:
Yes, the false is just to line up the other things. I'm assuming that compiler will optimize that out.
The single pipe is the way of saying that there's no need to perform short circuit evaluation on this.